packages/flutter_rx_bloc/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: flutter_rx_bloc
 description: Set of Flutter Widgets that help implementing the BLoC design pattern. Built to be used with the rx_bloc package.
-version: 6.1.0
+version: 6.2.0
```

Comment:
When upgrading dev dependencies we do not need to bump the version of the package